frontend/src/AlertsManagement/management_helpers.test.js

```diff
@@ -0,0 +1,120 @@
+import { getAlertsData, getAlertVisData } from "./management_helpers";
+import { enableFetchMocks } from 'jest-fetch-mock';
+enableFetchMocks();
+
+describe("getAlertsData", () => {
+  // Fake alert JSON data. 
+  const fakeAlerts = [{
+    anomalies: [
+      { dataPoints: {"2019-11-24": {value: 79}, }, 
+        dimensionName: "Ramen", metricName: "Interest Over Time",
```

Comment:
nit: `Ramen` and `Interest Over Time` can be `const` as used multiple times in this file

Reply:
same as prev comment: I would say that this isn't necessary and doesn't make as much sense to make them const because it's the mock data, but I can change it if you think that's better